### ISSUE TITLE
bookmarks disappear after beautify save 

### ISSUE THREAD

**Issue by ortiggs** [JeffryBooher/brackets-bookmarks-extension#27]
All the bookmarks disappear after beautifying or saving a file using Brackets Beautify (https://github.com/Hirse/brackets-beautify).

**Comment by JeffryBooher** [comment]
Yeah. It looks like some extensions are triggering a data change in codemirror which is removing the bookmarks data.  I will see if there is some fix but most-likely this is due to data-handling inside of codemirror by replacing the text content wholesale after running the code through a third-party service like beautify...
